### PR TITLE
Usergrid 347

### PR DESCRIPTION
**Pull request by GERey**
This merge includes the fixes in 493 and all of the tickets that fixed as well.

### CODE REVIEW

**Comment by GERey** [comment]
https://issues.apache.org/jira/browse/USERGRID-347

**Comment by GERey** [comment]
Also fixes https://issues.apache.org/jira/browse/USERGRID-338

**Comment by GERey** [comment]
This PR also fixes https://issues.apache.org/jira/browse/USERGRID-502

File: stack/rest/src/test/java/org/apache/usergrid/rest/management/ExportResourceIT.java [hunk 1]
```diff
@@ -60,734 +61,729 @@ public void exportCallSuccessful() throws Exception {
...
-        }
-        catch ( UniformInterfaceException uie ) {
-            responseStatus = uie.getResponse().getClientResponseStatus();
-        }
-        assertEquals( ClientResponse.Status.BAD_REQUEST, responseStatus );
     }
 
-
+//
+//    @Ignore( "is this test still valid knowing that the sch. won't run in intelliJ?" )
```
Comment (shawnfeldman) [inline stack/rest/src/test/java/org/apache/usergrid/rest/management/ExportResourceIT.java]: Why just comment out? can you just delete this?

**Pull request closed without merging**